project/Settings.scala

```diff
@@ -187,10 +187,16 @@ object Settings {
   )
 
   def slickGeneralSettings =
-    slickPublishSettings ++ slickScalacSettings ++ publishedScalaSettings ++ Seq(
+    slickPublishSettings ++ slickScalacSettings ++ scoverageSettings ++ publishedScalaSettings ++ Seq(
       logBuffered := false
     )
 
+  def scoverageSettings = Seq(
+    scoverage.ScoverageKeys.coverageMinimumStmtTotal := 58,
+    scoverage.ScoverageKeys.coverageFailOnMinimum    := true,
+    scoverage.ScoverageKeys.coverageExcludedFiles    := "<empty>;.*src_managed.*;.*slick-hikaricp.*;.*target.*;"
```

Comment:
What's the function of and reason for these exclusions?

Reply:
slick-hikaricp had a low coverage so, I ignore it as well as src_managed because of the custom settings used in `FMPP.scala`